bin/fuzz.py

```diff
@@ -185,27 +184,18 @@ def run(self):
         self.start_time = time.monotonic()
         self.iteration = 0
         self.tasks = 0
-        self.queue = parallel.Parallel(lambda task: task.run(bar), pin=True)
-
-        if self.queue.num_threads:
-            # pool of ids used for generators
-            self.tmp_ids = 2 * max(1, self.queue.num_threads) + 1
-            self.free_tmp_id = {*range(self.tmp_ids)}
-            self.tmp_id_count = [0] * self.tmp_ids
+        self.queue = parallel.create(lambda task: task.run(bar), pin=True)
```

Comment:
`create` is not a very descriptive name. Better `new` or `new_queue` or so

Reply:
Fair point, renamed to `new_queue` :slightly_smiling_face: